### ISSUE TITLE
Fails to build on FreeBSD

### ISSUE THREAD

**Issue by mneumann** [1Password/arboard#45]
```
error[E0432]: unresolved import `crate::common::ScopeGuard`
  --> src/x11_clipboard.rs:45:13
   |
45 | use crate::{common::ScopeGuard, common_linux::into_unknown, Error, LinuxClipboardKind};
   |             ^^^^^^^^^^^^^^^^^^ no `ScopeGuard` in `common`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `arboard` due to previous error
warning: build failed, waiting for other jobs to finish...
error: build failed
```

**Comment by complexspaces** [comment]
Sorry that this broke your code, I wasn't aware this crate had any usage on FreeBSD 😓.

**Comment by mneumann** [comment]
It's a dependency of `egui` :). Fixed by #46

**Comment by complexspaces** [comment]
This fix is now published in version 2.1.1.
